### ISSUE TITLE
Event timestamp is one hour off when requested as json

### ISSUE THREAD

**Issue by nurriz** [graphite-project/graphite-web#1281]
I noticed my events being off by an hour when I display them in grafana and initally thought the error to be there, but it seems the error is from graphite. When making an event I can see the right timestamp, but when reuested by /events/get_data the timestamp is an hour earlier than what's in the db.

![image](https://cloud.githubusercontent.com/assets/566325/8719400/ceb79940-2bab-11e5-8161-94fc6b58d75a.png)

This is the output I get from /events/get_data:
```json
[
  {
    "data": "",
    "what": "Test event 07/16/2015 09:37:43",
    "when": 1437028664.0,
    "id": 1360,
    "tags": "timetest"
  },
  {
    "data": "",
    "what": "Test event 07/16/2015 09:40:20",
    "when": 1437028820.0,
    "id": 1361,
    "tags": "timetest"
  }
]
```

This is what I see in the db:
![image](https://cloud.githubusercontent.com/assets/566325/8719478/4f6464e2-2bac-11e5-84d1-b64908b54036.png)

I am running 0.9.13 of both carbon and the webapp nistalled from pip. Django is 1.6.2. My timezone is set to Europe/Copenhagen.

**Comment by deniszh** [comment]
This problem was fixed in master but not in 0.9.x yet. Please check https://github.com/graphite-project/graphite-web/issues/1028, I'll try to fix that this weekend.

**Comment by dooblem** [comment]
Hello,
I have the problem too.
Some news on that ?
Thanks in advance,
Marc

**Comment by dooblem** [comment]
I just patched my 0.9.12 with the following to make it work:

```
--- views.py.bak	2015-09-15 16:23:13.000000000 +0200
+++ views.py	2015-09-15 16:25:55.000000000 +0200
@@ -7,7 +7,7 @@
 from django.shortcuts import render_to_response, get_object_or_404
 from pytz import timezone
 
-from graphite.util import json
+from graphite.util import json, epoch
 from graphite.events import models
 from graphite.render.attime import parseATTime
 
@@ -19,7 +19,7 @@
 class EventEncoder(json.JSONEncoder):
     def default(self, obj):
         if isinstance(obj, datetime.datetime):
-            return to_timestamp(obj)
+            return epoch(obj)
         return json.JSONEncoder.default(self, obj)
```

**Comment by obfuscurity** [comment]
Merged #1034 which should fix this in 0.9.x.
